docs/advanced_patterns.rst

```diff
@@ -83,11 +83,10 @@ This resolver can be used in a stack config file with the following syntax::
 
   template_path: <...>
   parameters:
-    param1:
-      <your resolver name>: <value>  # <your resolver name>
-                                     # should match Hook.name.
-                                     # <value> will be passed to the
-                                     # resolver's resolve() method.
+    param1: !<your_resolver_name>: <value> # <your resolver name> is the lower camel-case version
```

Comment:
this needs to be `!<your_resolver_name> <value>` - without the colon between resolver name and value